ui/src/app/components/runtime-configuration-panel.rtl.spec.tsx

```diff
@@ -1223,4 +1430,1154 @@ describe(RuntimeConfigurationPanel.name, () => {
     ).toBeInTheDocument();
     expect(screen.queryByText('Standard disk')).not.toBeInTheDocument();
   });
+
+  it('should allow configuration via GCE preset', async () => {
+    setCurrentRuntime(null);
+
+    const { container } = component();
+
+    const CustomizeButton = screen.getByRole('button', {
+      name: 'Customize',
+    });
+
+    await CustomizeButton.click();
+
+    // Ensure set the form to something non-standard to start
+    await pickMainCpu(container, 8);
+    await pickComputeType(container, ComputeType.Dataproc);
+    await pickStandardDiskSize(MIN_DISK_SIZE_GB + 10);
+
+    // GPU
+    await pickPresets(container, runtimePresets.generalAnalysis.displayName);
+
+    clickExpectedButton('Create');
+    await waitFor(() => {
+      expect(runtimeApiStub.runtime.status).toEqual('Creating');
```

Comment:
don't need to change it for this PR, but best practice is to include only 1 expect() in a waitFor() - then use expect() without waitFor() for the others.  the logic is that you have already waited at that point.

Reply:
i tried removing it here and all the other places